system76/gaze18/default.nix

```diff
@@ -6,7 +6,7 @@
     ../../common/gpu/nvidia/ampere
   ];
 
-  boot.initrd.kernelModules = [ "nvidia" ];
+  boot.initrd.kernelModules = [ "nvidia" "i915" "nvidia_modeset" "nvidia_drm" ];
```

Comment:
Could you add a comment here, how this fixes vulkan support. We had recently a discussion that about whether we should have nvidia in the initrd because it adds to the initrd size. So maybe it would be better if it would be made explicit here, that for this profile it's actually needed.